### PR TITLE
DM-32574: Make HexapodController compatible with ts_xml 10.1

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: doc/version_history.rst [hunk 1]
```diff
@@ -6,6 +6,20 @@
...
+
+* `HexapodCommander`: make compatible with ts_xml 10.1 (while retaining backwards compatibility).
+
+Requires:
+
+* ts_hexapod_controller 1.2.0
+* ts_hexrotcomm 0.23
+* ts_salobj 6.3
+* ts_idl 3.4
+* ts_xml 7.1
```
Comment (wvreeven) [inline doc/version_history.rst:20]: `7.1` -> `10.1`

**Review by wvreeven — APPROVED** [review]
Please modify the required ts_xml version. Apart from that this looks good to me.

File: doc/version_history.rst (hunk 1 above)
Comment (r-owen) [reply doc/version_history.rst:20]: This makes the package compatible with 10.1 *and* earlier versions.

Comment (wvreeven) [reply doc/version_history.rst:20]: Thanks for the clarification.

**Pull request merged**